src/main/resources/META-INF/NOTICE

```diff
@@ -12,58 +12,58 @@ Copyright 2014-2020 The Apache Software Foundation
 
 This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
 
-- com.amazonaws:amazon-kinesis-client:1.14.0
+- com.amazonaws:amazon-kinesis-client:1.11.2
```

Comment:
The versions in this file are not correct, we have updated some deps in our fork. Please correct, everything listed as "Included" should be in this file with the correct license type:

```[INFO] Excluding com.google.guava:guava:jar:18.0 from the shaded jar.
[INFO] Excluding commons-io:commons-io:jar:2.4 from the shaded jar.
[INFO] Excluding org.apache.commons:commons-lang3:jar:3.3.2 from the shaded jar.
[INFO] Including com.amazonaws:aws-java-sdk-kinesis:jar:1.11.844 in the shaded jar.
[INFO] Including com.amazonaws:aws-java-sdk-core:jar:1.11.844 in the shaded jar.
[INFO] Including software.amazon.ion:ion-java:jar:1.0.2 in the shaded jar.
[INFO] Excluding com.fasterxml.jackson.core:jackson-databind:jar:2.10.1 from the shaded jar.
[INFO] Excluding joda-time:joda-time:jar:2.5 from the shaded jar.
[INFO] Including com.amazonaws:jmespath-java:jar:1.11.844 in the shaded jar.
[INFO] Including com.amazonaws:aws-java-sdk-sts:jar:1.11.844 in the shaded jar.
[INFO] Including com.amazonaws:amazon-kinesis-producer:jar:0.14.0 in the shaded jar.
[INFO] Including com.google.protobuf:protobuf-java:jar:2.6.1 in the shaded jar.
[INFO] Excluding commons-lang:commons-lang:jar:2.6 from the shaded jar.
[INFO] Including com.amazonaws:amazon-kinesis-client:jar:1.14.0 in the shaded jar.
[INFO] Including com.amazonaws:aws-java-sdk-dynamodb:jar:1.11.844 in the shaded jar.
[INFO] Including com.amazonaws:aws-java-sdk-s3:jar:1.11.844 in the shaded jar.
[INFO] Including com.amazonaws:aws-java-sdk-kms:jar:1.11.844 in the shaded jar.
[INFO] Excluding commons-logging:commons-logging:jar:1.1.3 from the shaded jar.
[INFO] Including com.amazonaws:dynamodb-streams-kinesis-adapter:jar:1.5.0 in the shaded jar.
[INFO] Including software.amazon.awssdk:kinesis:jar:2.13.52 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-cbor-protocol:jar:2.13.52 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-json-protocol:jar:2.13.52 in the shaded jar.
[INFO] Including software.amazon.awssdk:protocol-core:jar:2.13.52 in the shaded jar.
[INFO] Including software.amazon.awssdk:profiles:jar:2.13.52 in the shaded jar.
[INFO] Including software.amazon.awssdk:sdk-core:jar:2.13.52 in the shaded jar.
[INFO] Including software.amazon.awssdk:auth:jar:2.13.52 in the shaded jar.
[INFO] Including software.amazon.eventstream:eventstream:jar:1.0.1 in the shaded jar.
[INFO] Including software.amazon.awssdk:http-client-spi:jar:2.13.52 in the shaded jar.
[INFO] Including software.amazon.awssdk:regions:jar:2.13.52 in the shaded jar.
[INFO] Excluding com.fasterxml.jackson.core:jackson-annotations:jar:2.10.1 from the shaded jar.
[INFO] Including software.amazon.awssdk:annotations:jar:2.13.52 in the shaded jar.
[INFO] Including software.amazon.awssdk:utils:jar:2.13.52 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-core:jar:2.13.52 in the shaded jar.
[INFO] Including software.amazon.awssdk:metrics-spi:jar:2.13.52 in the shaded jar.
[INFO] Including software.amazon.awssdk:apache-client:jar:2.13.52 in the shaded jar.
[INFO] Including software.amazon.awssdk:netty-nio-client:jar:2.13.52 in the shaded jar.
[INFO] Including io.netty:netty-codec-http:jar:4.1.46.Final in the shaded jar.
[INFO] Including io.netty:netty-codec-http2:jar:4.1.46.Final in the shaded jar.
[INFO] Including io.netty:netty-codec:jar:4.1.46.Final in the shaded jar.
[INFO] Including io.netty:netty-transport:jar:4.1.46.Final in the shaded jar.
[INFO] Including io.netty:netty-resolver:jar:4.1.46.Final in the shaded jar.
[INFO] Including io.netty:netty-common:jar:4.1.46.Final in the shaded jar.
[INFO] Including io.netty:netty-buffer:jar:4.1.46.Final in the shaded jar.
[INFO] Including io.netty:netty-handler:jar:4.1.46.Final in the shaded jar.
[INFO] Including io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.46.Final in the shaded jar.
[INFO] Including io.netty:netty-transport-native-unix-common:jar:4.1.46.Final in the shaded jar.
[INFO] Including com.typesafe.netty:netty-reactive-streams-http:jar:2.0.4 in the shaded jar.
[INFO] Including com.typesafe.netty:netty-reactive-streams:jar:2.0.4 in the shaded jar.
[INFO] Including org.reactivestreams:reactive-streams:jar:1.0.2 in the shaded jar.
[INFO] Including software.amazon.awssdk:sts:jar:2.13.52 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-query-protocol:jar:2.13.52 in the shaded jar.
[INFO] Including org.apache.httpcomponents:httpclient:jar:4.5.9 in the shaded jar.
[INFO] Excluding commons-codec:commons-codec:jar:1.10 from the shaded jar.
[INFO] Including org.apache.httpcomponents:httpcore:jar:4.4.11 in the shaded jar.
[INFO] Excluding com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:jar:2.10.1 from the shaded jar.
[INFO] Excluding com.fasterxml.jackson.core:jackson-core:jar:2.10.1 from the shaded jar.
[INFO] Excluding org.apache.flink:force-shading:jar:1.11.2 from the shaded jar.```